commit 8754a4133e154ca853e6765a3fe5c7a904c77626
Author: Joseph Myers <joseph@codesourcery.com>
Date:   Fri May 26 15:03:31 2023 +0000

    Add AT_RSEQ_* from Linux 6.3 to elf.h
    
    Linux 6.3 adds constants AT_RSEQ_FEATURE_SIZE and AT_RSEQ_ALIGN; add
    them to glibc's elf.h.  (Recall that, although elf.h is a
    system-independent header, so far we've put AT_* constants there even
    if Linux-specific, as discussed in bug 15794.  So rather than making
    any attempt to fix that issue, the new constants are just added there
    alongside the existing ones.)
    
    Tested for x86_64.

diff --git a/elf/elf.h b/elf/elf.h
index 4738dfa28f6549fc..076d8e3f696c58f7 100644
--- a/elf/elf.h
+++ b/elf/elf.h
@@ -1205,6 +1205,9 @@ typedef struct
 #define AT_HWCAP2	26		/* More machine-dependent hints about
 					   processor capabilities.  */
 
+#define AT_RSEQ_FEATURE_SIZE	27	/* rseq supported feature size.  */
+#define AT_RSEQ_ALIGN	28		/* rseq allocation alignment.  */
+
 #define AT_EXECFN	31		/* Filename of executable.  */
 
 /* Pointer to the global system page used for system calls and other
